documentation/bolt_installing_modules.md

```diff
@@ -1,14 +1,51 @@
-# Modules
-
-Tasks and plans are packaged in Puppet modules, so you can install them as you would any module and manage them with a Puppetfile. 
+# Installing modules
+
+To share the Bolt plans and tasks that you've created on the Forge, you can
+package them into Puppet modules. If you'd like to use a plan or task from a
+module that you found on the Forge, you can use Bolt to install the module.
+
+## Install a module
+
+> **Before you begin**
+> 
+> - In your Bolt project directory, create a file named `Puppetfile`. 
+> - Add any task or plan modules stored locally in `modules/` to the list. For
```

Comment:
```suggestion
> - Add any task or plan modules stored locally in `modules/` to the list. For
```
@lucywyman I just noticed this. Do you think it's better to say "Add any tasks or plans stored in..." or "Add any module content stored in..."

Reply:
Maybe just 'Add any modules stored locally' or 'Add any Puppet modules stored locally'? Now that plan and task content can be at the top-level directory, I don't expect as many people to get started with Bolt by manually creating a module in `modules/`.